install/helm/openchoreo-observability-plane/templates/prometheus/servicemonitor-openchoreo-controllers.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openchoreo-controllers
```

Comment:
Is this only meant for single-cluster installations? How does this query the control-plane, when it is in a different cluster?